packages/sit-onyx/src/components/OnyxListbox/OnyxListbox.stories.ts

```diff
@@ -22,8 +23,42 @@ const meta: Meta<typeof OnyxListbox> = {
   title: "support/OnyxListbox",
   ...defineStorybookActionsAndVModels({
     component: OnyxListbox,
-    events: ["update:modelValue"],
+    events: ["update:modelValue", "loadMore"],
+    argTypes: {
```

Comment:
<img width="907" alt="image" src="https://github.com/SchwarzIT/onyx/assets/151019977/29bc5785-5622-46a7-a900-01c39227ae79">

maybe deactivate the control for the slot?